### PR TITLE
Finished bill creation page and added proxy to bypass CORS

### LINKED ISSUE #33: Feature Request: Add a bill
**User Story**
As a user
I want to be able to add a bill
So I can record who owes whom

**Priority**
[x]: High
[]: Medium
[]: Low

**Describe the solution you want to implement**
A page for adding a new bill. Bills have a title, you can add diners to a bill, each diner has a name, and you can define which diners have paid the bill already.  The user has to enter the total bill amount. The user can return back to the homepage if they decide to cancel adding a bill.

**Describe alternatives you've considered**
Having a field for a total bill on the next screen in a "What is the total bill? __" form. This facilitates work split, however it is much more logical to have all bill info fields on one screen/page.
**Issue Dependencies**
#13 
**Additional context**
At this point the issue does not need to be tied to the back-end, however it needs to be developed with foresight for easy integration of back-end.

### PR DESCRIPTION
**Pull request by jake-good**
Closes #33 

**Description**

Made final additions to bill creation page so now users can enter a title an amount and select a payer. Added proxy to package.json to ensure POST to API can bypass CORS policy

**Testing**

Manually tested interface and debugged POST request to ensure correct JSON object sent.

**Checklist**

- [x] I have tested my change
- [x] Code builds successfully and all tests pass
- [ ] Docs have been added/updated if needed

### CODE REVIEW

File: split-webapp/split/src/pages/Split.js [hunk 1]
```diff
@@ -28,33 +33,44 @@ class Split extends Component {
...
     });
   }
 
+  handleChange(name) {
+    const { transaction } = this.state;
+
+    this.setState({
+      transaction: {
+        ...transaction,
+        payed: name
```
Comment (vee-christella) [inline split-webapp/split/src/pages/Split.js:48]: Nit: paid? 😛

Comment (vee-christella) [reply split-webapp/split/src/pages/Split.js:48]: Maybe `paidBy` because `paid` implies a boolean rather than a person

**Review by vee-christella — APPROVED** [review]
LGTM!

**Review by williamlixu — APPROVED** [review]
LGTM!

**Pull request merged**